Source/Core/Core/HW/GCMemcard/GCIFile.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include "Common/CommonTypes.h"
+#include "GCMemcard.h"
```

Comment:
Include should be relative:

```cpp
#include "Core/HW/GCMemcard/GCMemcard.h"
```

Reply:
Even if both files are in the same folder? But sure, not problem.